pkg/cortex/provider.go

```diff
@@ -0,0 +1,23 @@
+package cortex
+
+import "github.com/grafana/grizzly/pkg/grizzly"
+
+// Provider defines a Cortex Provider
+type Provider struct{}
+
+// NewProvider returns a new Cortex Provider
+func NewProvider() *Provider {
+	return &Provider{}
+}
+
+// GetName returns the name of the Cortex provider
+func (p *Provider) GetName() string {
+	return "cortex"
```

Comment:
This looks like a provider to talk to Hosted Prometheus, maybe name it as such?

Reply:
done, see below.